### ISSUE TITLE
Jenkins error on getting absolute path of sahagin.yml with Linux master and Windows slave.

### ISSUE THREAD

**Issue by hiroshitoda** [SahaginOrg/sahagin-java#6]
With Jenkins Linux master and Windows slave, getting absolute path of sahagin.yml fails as below:

```

Tests run: 1, Failures: 0, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:44 min
[INFO] Finished at: 2015-03-16T11:22:35+09:00
[INFO] Final Memory: 11M/179M
[INFO] ------------------------------------------------------------------------
java.lang.IllegalArgumentException: config file not found: /C:\Jenkins\workspace\FooBarProject\sahagin.yml
    at org.sahagin.main.SahaginMain.main(SahaginMain.java:61)
    at org.sahagin.jenkins.SahaginReportPublisher.perform(SahaginReportPublisher.java:57)
    at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
    at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:761)
    at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:721)
    at hudson.model.Build$BuildExecution.post2(Build.java:183)
    at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:670)
    at hudson.model.Run.execute(Run.java:1775)
    at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
    at hudson.model.ResourceController.execute(ResourceController.java:89)
    at hudson.model.Executor.run(Executor.java:240)
Build step 'Publish Sahagin HTML report' marked build as failure
成果物を保存中
ERROR: 指定されたファイルパターン「sahagin-report」に合致するファイルがありません。設定ミス？
Build step '成果物を保存' changed build result to FAILURE
Finished: FAILURE

```

It seems that path of "config file" is not treated as Windows path.

That exception may be thrown here:
https://github.com/SahaginOrg/sahagin-java/blob/bbfdb0af28e721518786668e1062ab1ee2a49a4b/src/main/java/org/sahagin/main/SahaginMain.java#L63

**Comment by hiroshitoda** [comment]
Similar error occurs with Windows master and Windows slave. Only one difference at top of stacktrace:

```
java.lang.IllegalArgumentException: config file not found: C:\Jenkins\workspace\FooBarProject\sahagin.yml
```

I guess that SahaginMain.main(String[] args) may be called on Jenkins master, not slave.

**Comment by NozomiIto** [comment]
You set relative path to the "Sahagin config YAML" setting in Jenkins project post-build action?

Sahagin Jenkins plug-in calculates config YAML absolute path from "Sahagin config YAML" value by getRemote method here:
https://github.com/SahaginOrg/sahagin-jenkins-plugin/blob/master/src/main/java/org/sahagin/jenkins/SahaginReportPublisher.java#L57

This logic may cause problem..

**Comment by NozomiIto** [comment]
As you mentioned, SahaginMain is called on Jenkins master.

I fixed this problem, and now you can download the latest hpi file from
http://maven.jenkins-ci.org:8081/content/repositories/snapshots/org/jenkins-ci/plugins/sahagin/0.6.1-SNAPSHOT/sahagin-0.6.1-20150320.153428-6.hpi

Can you install this hpi file to your Jenkins, and try it?

**Comment by NozomiIto** [comment]
You need to use this plug-in with Sahagin jar version 0.6.

**Comment by hiroshitoda** [comment]
The latest plugin(0.6.1) works collectly with Jenkins slave. Thanks.

**Comment by NozomiIto** [comment]
Released [Sahagin 0.7](https://github.com/SahaginOrg/sahagin-java/releases/tag/0.7) which resolves this   issue. Please use it.
